### PR TITLE
Create tasks_layout directory while packaging

### PR DESCRIPTION
**Pull request by issacnitin**

### CODE REVIEW

File: ci/ci-util.js [hunk 1]
```diff
@@ -391,6 +391,7 @@ var createTasksZip = function () {
     // zip the tasks
     console.log();
     console.log('> Zipping the tasks')
+    fs.mkdirSync(tasksLayoutPath);
```
Comment (damccorm) [inline ci/ci-util.js:394]: This change is confusing to me - why are we creating the tasks here? Shouldn't they already exist in this path if we're trying to zip them up?

Could you share a link to the failing build?

**Comment by damccorm** [comment]
I think this is the wrong thing to do. I think we want this pr from me - https://github.com/microsoft/azure-pipelines-tasks/pull/12968

File: ci/ci-util.js (hunk 1 above)
Comment (issacnitin) [reply ci/ci-util.js:394]: @damccorm You can find failing build here https://mseng.visualstudio.com/PipelineTools/_build/results?buildId=12183622&view=logs&j=3e13d126-0b89-5035-9868-7a56405ed192&t=2cc64d6b-65bf-59c2-2d25-6673fbda3f9e

Comment (issacnitin) [reply ci/ci-util.js:394]: In line 369, we're creating the directory for the other function, similarly, we need to create here also

**Comment by issacnitin** [comment]
@damccorm I shall close the PR. you can reopen and merge if you think this is useful.

**Pull request closed without merging**